src/icons/two-toned/IconAnnotation.tsx

```diff
@@ -13,7 +13,15 @@ const IconAnnotation = ({ className = '', height = 24, title, width = 24 }: TwoT
         width={width}
     >
         <g fillRule="nonzero" fill="none">
-            <circle className="background-color" fill={bdlYellorange} cx="16.162" cy="16.162" r="16" />
+            <circle
+                aria-label="Inline comment"
```

Comment:
this seems like specific text for an icon that could have multiple use cases. for example, Notes or Canvas could be using this icon in a way that's not related to inline comments

could this be handled in a more general way?